### DISCUSSION TITLE
GitHub copilot on premises

### DISCUSSION THREAD

**Discussion by Dyccl** [community/community#38858] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,
I absolutely love GitHub copilot.
Was wondering if there is or will be an “on prem” version for companies to purchase ?

**Comment by davecheney** [comment, marked as answer]
I’m sorry there are no plans for an on premise version of copilot.

**Reply by dvirsegal** [reply]
Now that it is 2023, are there any plans for an on-premise version?

**Reply by davecheney** [reply]
There are no plans for an on premise version of Copilot.
